src/main/java/com/infobip/interviewdemo/domain/CommentType.java

```diff
@@ -0,0 +1,7 @@
+package com.infobip.interviewdemo.domain;
```

Comment:
```suggestion
package com.infobip.interviewdemo.enums;
```
i think this would be more suitable for it